web/source/keymanweb.js

```diff
@@ -3003,17 +3023,26 @@ if(!window['keyman']['initialized']) {
     keymanweb._FindCaret = function(Pelem)     // I779
     {
       if(!Pelem.createTextRange && Pelem.selectionStart)
-      {      
-        var Levent=document.createEvent('KeyboardEvent');
-        if(Levent.initKeyEvent)
-        {
-          Levent.initKeyEvent('keypress',true,true,null,false,false,false,false,0,32);
+      {
+        var Levent = new KeyboardEvent('keypress', {
+          "code":"Space",
```

Comment:
I don't think this is going to work on IE. We need to stay with `initKeyEvent`. There's no benefit to two code paths here.

Reply:
Good, that's fine.  This was always a Firefox-oriented fix - it's not supposed to run on IE.

Reply:
Also, I'm in the process of investigating whether we can outright drop the function containing this code - `_FindCaret` is explicitly a workaround for an old Firefox bug, so it's probably no longer something we need in KMW.